src/test/resources/general.properties

```diff
@@ -1,8 +1,8 @@
 # target execution: local, selenium-grid or testcontainers
-target = local
+target = testcontainers
 
 # browser to use for local and testcontainers execution
-browser = chrome
+browser = edge
```

Comment:
Could you revert the change to the original value?

Reply:
done